CollAction/GraphQl/Queries/CrowdactionGraph.cs

```diff
@@ -98,6 +104,23 @@ public CrowdactionGraph(IEfGraphQLService<ApplicationDbContext> entityFrameworkG
 
                     return c.Source.Percentage;
                 });
+            FieldAsync<NonNullGraphType<ListGraphType<NonNullGraphType<InstagramWallItemGraph>>>, IEnumerable<InstagramWallItem>>(
+                "instagramWall",
+                resolve: c =>
+                {
+                    string? instagramUser = c.Source.InstagramUser;
+                    if (instagramUser != null)
+                    {
+                        return c.GetUserContext()
+                                .ServiceProvider
+                                .GetRequiredService<IInstagramService>()
+                                .GetItems(instagramUser, c.CancellationToken);
+                    }
+                    else
+                    {
+                        return Task.FromResult(Enumerable.Empty<InstagramWallItem>());
```

Comment:
Why not use async / await delegate here?

Reply:
Because it adds overhead. Just straight-up passing the task object down is a lot more efficient.